tests/unittests/NNPIOptPipelineTest.cpp

```diff
@@ -415,6 +422,7 @@ static void setupSplitParallelizationTestFCReluNNPI(
   nodeInfo[TN]["NNPI_extraEdges"].push_back(FC->getName().str() + "@7");
   nodeInfo[RN]["NNPI_extraEdges"].push_back(SN->getName().str());
   nodeInfo[TN]["NNPI_extraEdges"].push_back(SN->getName().str());
+  nodeInfo[AN]["NNPI_extraEdges"].push_back(TN->getName().str());
```

Comment:
@jsubag This edge does not appear to be respected in the HW graph final solution.